index.js

```diff
@@ -203,8 +212,8 @@ Point.prototype = {
     },
 
     _mult: function(k) {
```

Comment:
I'm sorry, I just add commit to my forked version and not find out that in will occur in pull request. I'll correct it.